wrapper/src/main/java/software/amazon/jdbc/PropertyDefinition.java

```diff
@@ -156,6 +156,16 @@ public class PropertyDefinition {
               "true", "false"
           });
 
+  public static final AwsWrapperProperty ENABLE_GREEN_NODE_REPLACEMENT =
+      new AwsWrapperProperty(
+          "enableGreenNodeReplacement",
+          "false",
+          "Enables to replace a green node host names with original host name when green host DNS doesn't exist.",
```

Comment:
Enables replacing a green node host name with the original hostname after a blue/green switchover and the green name no longer resolves